### DISCUSSION TITLE
MacOS Desktop Version Controls

### DISCUSSION THREAD

**Discussion by jpinacate** [organicmaps/organicmaps#6225] [Q&A] [answered]
I'm running the desktop version on MacOS with a TrackPad. I see some commands require "Alt + Right mouse button". How do you generate this command on a Mac with TrackPad (or touchpad)? 

I've googled and found lots of suggestions but none seem to work. Same question for 'Left mouse button'. 

Thanks!

**Comment by biodranik** [comment, marked as answer]
Alt + two fingers tap?

**Reply by jpinacate** [reply]
Thanks! How about the Left Mouse Click?

**Reply by biodranik** [reply]
Alt + single tap?

**Reply by jpinacate** [reply]
In MacOS Settings (Settings->Trackpad->Point & Click) see item 'Secondary click'. This item effectively refers to 'Right Mouse Click'. In my case it was turned off I set it to 'Two finger tap' and all was well.
Thanks for the help!
